### DISCUSSION TITLE
Could anyone please tell me something about creating radar charts using plotly on Redash?

### DISCUSSION THREAD

**Discussion by qq20230530** [getredash/redash#6145] [Q&A]
I do not really understand 'x' and 'ys'.

This is what is written in the "custom code" panel:
// Available variables are x, ys, element, and Plotly
// Type console.log(x, ys); for more info about x and ys
// To plot your graph call Plotly.plot(element, ...)
// Plotly examples and docs: https://plot.ly/javascript/

It says: // Type console.log(x, ys); for more info about x and ys
But where should I type "console.log(x, ys);" to get the info about x and ys???

**Comment by Auric-Manteo** [comment]
You are supposed to type it in the custom code textfield.
I only see that x is an empty array and ys an object, though. So not very helpful yet.

**Comment by Auric-Manteo** [comment]
Ah, got it. You need to set the X Column and Y Column above. Then x is an array of x values and y is object with an attribute that has the same column name you picked for the y column which contains an array of the y values. 🎉
